graph.go

```diff
@@ -121,7 +121,9 @@ func (g *Graph) AddTables(groupSize int, tables []*spansql.CreateTable) error {
 		}
 		for _, c := range t.Constraints {
 			opt := make(map[string]string)
-			opt["label"] = c.Name
+			if c.Name != "" {
+				opt["label"] = c.Name
```

Comment:
could you change to use comma joined `c.Columns` ?
https://github.com/googleapis/google-cloud-go/blob/spanner/v1.3.0/spanner/spansql/types.go#L214

Reply:
Sorry, I didn't catch it. Do you propose to generate the constraint name for the foreign key as column names joined by a comma? I thought it could be not a bad idea but, you see, the constraint has no "official name". I decided just to fix an exception with minimum efforts then discuss with you - it is acceptable at all to provide some name when it missed? And what is the best way to mark that the constraint name is generated but not provided by a schema file?
My goal was mainly to clarify the line of code that causes the exception. I wrote the Go code first time in my life there :)

Reply:
> . I decided just to fix an exception with minimum efforts then discuss with you

okay

>  Do you propose to generate the constraint name for the foreign key as column names joined by a comma? 
> the constraint has no "official name".


As mentioned above, constraint doesn't have an official name, so I suggested naming the edge by the column of the constraint's referenced table, but that may not be the best.